src/algebra/monoid_algebra/basic.lean

```diff
@@ -366,6 +372,10 @@ variables (k G)
   map_one' := rfl,
   .. of_magma k G }
 
+@[simp] lemma comp_of_hom_apply {H : Type*} [mul_one_class G] [mul_one_class H]
```

Comment:
Can't `simp` already prove this? This lemma can probably be removed

Reply:
You are right: `by simp` proves it.  The squeeze is
```lean
by simp only [monoid_hom.coe_comp, function.comp_app, of_apply]
```
I removed the lemma.